test/reflector.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
```

Comment:
I can't see any of this code in the output. Is it just a stub? Maybe we should remove it or make it explicitly a stub?

Reply:
You can't "see" it, as the new output is identical to the old output, so there are no diff in the snapshots.

Reply:
The only diff is for tests without processors.

Reply:
Ah, yes. That's the confusion.